src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
         // create three balls using class Lottery and print information about them in console
+        int ballsCount = 3;
```

Comment:
Consider replacing the magic number `3` with a named constant. This will improve the readability and maintainability of your code. For example, you could define a constant `private static final int NUMBER_OF_BALLS = 3;` and use it here.